### PR TITLE
Ticket/improve s3 replication doc

### PR DESCRIPTION
**Pull request by david-richer-adsk**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/f5b25168632762491030e7165ffcab13bb41ede6/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/932712471f476f12406528aacd31f0254c489ace/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/0b04fcb131fd2a444d0b3ae00729182390ef7179/index.html)

File: docs/en/trusted-solutions/tier1/setup/s3_replication.md [hunk 1]
```diff
@@ -15,26 +15,33 @@ It's possible to add S3 replication between two S3 buckets in different regions
...
+ * Configurable delay for new media to be replicated before being made available to users
 
 # Setup steps
 
-  * Create the replica S3 bucket in a new AWS region
-  * Update your Shotgun role policy to allow Shotgun to access the replica bucket
-  * Setup the replication rules on the primary S3 bucket
-  * Setup a VPC + Direct Connect + S3 proxy in the new AWS region
-  * Contact Shotgun Support to configure your site to use the new S3 replica bucket, providing the following:
+  * Create the replica S3 bucket in a new AWS region. See [Private S3 Bucket](../s3_bucket.md)
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/setup/s3_replication.md:40]: Broken link, file exists in same path, not parent.

File: docs/en/trusted-solutions/tier1/setup/s3_replication.md [hunk 2]
```diff
@@ -15,26 +15,33 @@ It's possible to add S3 replication between two S3 buckets in different regions
...
+  * Update your existing Shotgun role policy to allow Shotgun to also access the replica bucket
+  * Setup the replication rules on the primary S3 bucket. See [How do I add a replication rule to an S3 bucket?](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/enable-replication.html#enable-replication-add-rule)
+  * Setup a VPC + Direct Connect + S3 proxy in the new AWS region. See [Media Segregation](../media_segregation.md)
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/setup/s3_replication.md:43]: Broken link, file exists in same path, not parent.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/4766b851c4892b86b865f5e082b5b55702fd2bc1/index.html)

**Review by rlyne — APPROVED** [review]

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/9a2459f4a74317edcf16066e79ddc19844154f9d/index.html)

**Pull request merged**